app/views/fragments/checkout/awinPixel.scala.html

```diff
@@ -0,0 +1,16 @@
+@import com.gu.memsub.subsv2.Subscription
+@import com.gu.memsub.subsv2.SubscriptionPlan.ContentSubscription
+@import model.SubscriptionOps._
+@import views.support.PlanOps._
+@import com.gu.memsub.promo.Promotion.AnyPromotion
+
+@(subscription: Subscription[ContentSubscription], promotion: Option[AnyPromotion] = None)
+<!-- Image Pixel Tracking - Mandatory -->
+<img src="https://www.awin1.com/sread.img?tt=ns&tv=2
+&merchant=11152
+&amount=@subscription.firstPrice
+&cr=@subscription.currency
+&ref=@subscription.id.get
```

Comment:
Is `id` an option, should this have a getOrElse?

Reply:
It is not an option actually it just has a val called get

Reply:
Fair enough, slightly unexpected!